CONTRIBUTING.md

```diff
@@ -151,6 +152,139 @@ Validate authentication using the username `alice` with the password `alice`:
 $ curl -v -k -X POST -d "alice" http://localhost:3000/authn-ldap/test/cucumber/alice/authenticate
 ```
 
+
+
+#### Google Cloud Platform (GCP) Authentication
+
+To enable a host to log into Conjur using GCP identity token, run `start` with the `--authn-ldap` flag and pass a name of a running Google Compute Engine (GCE).
+
+
+
+**Prerequisites**
+
+- A Google Cloud Platform account. https://cloud.google.com/.
+- An access to a running Google Compute Engine instance. 
+
+
+
+**Obtaining the Instance Identity Token**
+
+An identity token is obtained from the Google's metadata server. [(Refer to Google Cloud documentation on obtaining the instance identity token).](https://cloud.google.com/compute/docs/instances/verifying-instance-identity#request_signature)  To access Google's metadata server you need to run a *curl* command inside a running GCE instance.
```

Comment:
It is more readable if the whole sentence was not a link.

An identity token is obtained from the Google's metadata server. Refer to Google Cloud [documentation](https://cloud.google.com/compute/docs/instances/verifying-instance-identity#request_signature) on obtaining the instance identity token. To access Google's metadata server you need to run a *curl* command inside a running GCE instance.